src/sentry/models/commitauthor.py

```diff
@@ -11,11 +11,15 @@ class CommitAuthor(Model):
     organization_id = BoundedPositiveIntegerField(db_index=True)
     name = models.CharField(max_length=128, null=True)
     email = models.EmailField()
+    external_id = models.CharField(max_length=164, null=True, db_index=True)
```

Comment:
We don't need `db_index` here. This will create an extra index that we don't need. The `UNIQUE` is enough.